backend/src/main/java/ca/bc/gov/hlth/hnweb/service/EligibilityService.java

```diff
@@ -0,0 +1,30 @@
+package ca.bc.gov.hlth.hnweb.service;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.time.DateUtils;
+import org.springframework.stereotype.Service;
+
+import ca.bc.gov.hlth.hnweb.model.CheckEligibilityResponse;
+
+@Service
+public class EligibilityService {
+	
+	/**
+	 * TODO (weskubo-cgi) This is just a stubbed out service for unit testing.
+	 * The real service will integrated with a downstream application and return
+	 * HL7 objects.
+	 * @return
+	 */
+	public CheckEligibilityResponse checkEligibility(String phn, Date eligibilityDate) {
```

Comment:
Param eligibilityDate is not currently used. Might be needed in future implementation, check if we want to keep it for now.

Reply:
It will be require once we integrate the downstream service.